app/views/Layout.scala.html

```diff
@@ -35,10 +37,13 @@
         deskproServiceName: Option[String] = None,
         serviceTitle: Option[String] = None,
         additionalScripts: Option[Html] = None,
-        headBlock: Option[Html] = None
+        headBlock: Option[Html] = None,
+        journey: Option[Journey] = None
 )(contentBlock: Html)(implicit request: RequestHeader, messages: Messages)
 
-
+@forceNavigation = @{
+    journey.exists(appConfig.forceServiceNavigation)
+}
```

Comment:
Not sure this makes sense; as we're checking if within a type of `Option[Journey]` whether there exists a `true` or `false`?

I think we can delete this block, and just use the `appConfig.forceServiceNavigation`?